.github/workflows/build-workflow.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           fetch-depth: 0
 
       - name: Enable toolchain via github action
-        uses: dtolnay/rust-toolchain@1.63.0
+        uses: dtolnay/rust-toolchain@1.64.0
```

Comment:
I wish I could define the version numbers of all these toolchains spread across all these workflows at a single place. If anyone knows some trick, let me know.

Reply:
We can try this feature together. https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows